### ISSUE TITLE
Can't run locally after npm install, Gulp Serve

### ISSUE THREAD

**Issue by Jdtanacredi** [josephdburdick/adoptive-2015#1]
Having trouble running on my local machine. I cloned repo, ran npm install, gulp serve, but gulpfile uses babel which I am unfamiliar with. Please advise and I'll add pull request to readme.

**Comment by josephdburdick** [comment]
Thanks for leaving an issue. First off, try `$ node -v`. If it returns any version below 4 you should install the new version of Node from [here](https://nodejs.org). Try the following (from the project root directory):

```
$ rm -rf ./node_modules
$ (sudo) npm install -g babel
$ npm install
$ gulp serve
```

If this fails please post here the error stack trace message you receive and we'll resolve this together.

**Comment by Jdtanacredi** [comment]
Error listed below:

[12:03:33] No gulpfile found 

Babel installed, do I have to run anything to parse the gulpfile.babel.js ?

**Comment by josephdburdick** [comment]
What does `$ node -v` return?

**Comment by Jdtanacredi** [comment]
I'm on v4.0.0

**Comment by josephdburdick** [comment]
Ohhh, I think I know what it is. What does `$ gulp -v` return?

Mine returns the following:

```
[12:08:59] CLI version 3.9.0
[12:08:59] Local version 3.9.0
```

**Comment by josephdburdick** [comment]
I think you may need to:

```
npm uninstall gulp -g
npm install gulp -g
```

**Comment by josephdburdick** [comment]
Resolved in #2 .
